### DISCUSSION TITLE
WLED RPM LEDs

### DISCUSSION THREAD

**Discussion by pnbr0014** [sjdawson/GentlemanDriverPlugin#15] [Q&A]
How possible would it be to make a rev strip using your method of wled integration? Can you do anything with the game telemetry and wled?

**Comment by sjdawson** [comment]
This is certainly something I want to work towards. The next release of this plugin will change from the current "send JSON to WLED" to using WLED's live protocols to enable better, consistent, updating of the LEDs. I need to work out how I can possibly grab an existing LED configuration from SimHub to pass to WLED. 👍🏻

**Reply by pnbr0014** [reply]
I actually came across another plugin that does this but the issue is that you can't customize colors. You can configure number of LEDs etc and the shift point etc but it's very limited. Have a look!

https://github.com/hxlcyxn/SimHub-WLED

This might give you a head start
